src/www/ui_components/add-server-view.js

```diff
@@ -165,8 +165,14 @@ Polymer({
       <div class="footer center top-divider">
         <div
           id="addServerFooter"
+          hidden$="[[showAltAccessMessage]]"
           inner-h-t-m-l="[[localize('server-create-your-own', 'breakLine', '<br/>', 'openLink', '<a href=https://s3.amazonaws.com/outline-vpn/index.html>', 'closeLink', '</a>')]]"
         ></div>
+        <div
+          id="addServerFooterAlt"
+          hidden$="[[!showAltAccessMessage]]"
```

Comment:
This approach is problematic because the both templates will be rendered. It pollutes the DOM, causes subtle bugs if you reuse ids and one of them may crash with missing messages.

Use <dom-if> instead: https://polymer-library.polymer-project.org/3.0/docs/devguide/templates#dom-if

Reply:
After chatting with @jyyi1  I switched this to use dom-if in `servers_view/index.ts` but not here. `_inputInvalidChanged` here is already relying on the `hidden` property in an inadvisable way. But we can't stop using it here without breaking or entirely refactoring that code.

Reply:
Welcome to developing in the client @ohnorobo !!!!

Reply:
@fortuna I have a proposal in my comment below, which can simplify the logic in `_inputInvalidChanged`. And you will also be able to use `dom-if` as well because we will not reference any specific elements in the function. But it requires more code changes. Depending on the workload, we can either refactor it later or do it now.

Reply:
done